### ISSUE TITLE
FeedbackChallenge : WooCommerce & Other ecom integratio

### ISSUE THREAD

**Issue by stephen-rowan** [Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal#46]
## FeedbackChallenge : [WooCommerce & Other ecom integratio](https://cardano.ideascale.com/a/dtd/WooCommerce-Other-ecom-integratio/351487-48088)

In the spirit of Project Catalyst's #FeedbackChallenge we are assessing to what extent the Catalyst Funding Process and it's Proposals are **open source** as part of the proposal [Quality-Assurance-DAO](https://cardano.ideascale.com/a/dtd/Quality-Assurance-DAO/351678-48088). Your proposal has been selected because "open source" is mentioned in it's text or comments.

We would appreciate it if you could respond to the following questions (in the reply to my comments below) : 

1.  **Open** - Is your Organisation open-source ?
> We've simplified the solution and have made it completely open-source licenses under Apache 2.0.
2.  **Accessible** -  Can community contributors issue pull requests or raise issues on your source code ?
> We've added detail in regards to how we are an open-source development company, how users can contribute code or perform a PR and submit support requests via GitHub or a support forum. We may simply use WordPress's support forum area for each plugin that is submitted to the plugin repository.
3.  **Transparent** - Can community contributors access and propose changes to the source text of your documentation ?
> Documentation will be contained in GitHub and published to a static site supporting the plugin. Users can provide pull requests to add, adjust and contribute to the documentation.

To keep the assessment transparent each comment is recorded as a [GitHub Issue]( https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/issues/46 ) on **Quality-Assurance-DAO's** proposal repository (Apache-2.0 License) where a more detailed explanation can be found. 

Together we can build a community of open source contributors on the Cardano platform.

Thank You 

Stephen Whitenstall, QA-DAO

## Detailed explanation 

Your [Issue reference](https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/issues/46) includes a more detailed explanation of the methodology of this FeedbackChallenge : Open Source. There is also a *Open Source Assessment* [Project Board](https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/projects/3) and a GitHub bot post Issues in my channel #quality-assurance-dao on Project Catalyst's Discord.

We believe this is important because many developers are concerned with the lack of a clear commitment to open source and fear their free contributions will be locked into a proprietary platform.

## Open Source Assessment Method
- [x] Draft a basic Open Source Assessment Method.
https://github.com/Quality-Assurance-DAO/DAO-Open-Source/tree/main/Documents/Assessments

### Basic Open Source Assessment
1.  **Open** - Is your Organisation open-source ?
1.  **Accessible** -  Can your users issue pull requests or raise issues on your organisation's source code ?
1.  **Transparent** -  Can your users access and propose changes to the source of your organisation's documentation ?

### Method
- [x] Do text search of "open source" on Catalyst Projects.
- [x] Post assessment questions in comments of proposal.
- [x] Raise Issue for each proposal's assessment.
- [x] Record response to assessment.
- [ ] Close Issue once proposal submitted and record responses

### Report
- [ ] Collate and publish report of findings with a Creative Commons License.

**Comment by stephen-rowan** [comment]
> Peter (@astroboysoup) kudos icon
>Idea Submitter

> swhitenstall Thank you for bringing that to my attention.

> I have revised the proposal after talking to some other developers and community members.

> We've simplified the solution and have made it completely open-source licenses under Apache 2.0.

> We've added detail in regards to how we are an open-source development company, how users can contribute code or perform a PR and submit support requests via GitHub or a support forum. We may simply use WordPress's support forum area for each plugin that is submitted to the plugin repository.

> Documentation will be contained in GitHub and published to a static site supporting the plugin. Users can provide pull requests to add, adjust and contribute to the documentation.

**Comment by stephen-rowan** [comment]
Thank you for your response, Peter (astroboysoup). It has helped a lot.

I have captured them at [ https://github.com/Quality-Assurance-DAO/F5-Developer-ecosystem-Proposal/issues/46 ] and moved the status of the Issue to "Done".

Note : The Issue can still be altered and commented upon via GitHub until Fund 5 Proposals are finalized.

As I set out in the "Detailed explanation" section of each Issue : I will collect all data in a report of findings to be published on GitHub with a Creative Commons License after Fund 5 Proposals are finalized.

Good Luck with your proposal.

Stephen Whitenstall QA-DAO
